tests/lib/rules/computed-property-getters.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/computed-property-getters');
+const RuleTester = require('eslint').RuleTester;
+
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester();
+const parserOptions = { ecmaVersion: 2018, sourceType: 'module' };
+
+const codeWithoutGetters = [
+  `
+  {
+    foo: computed('model', function() {})
+  }`,
+  `{
+      foo: computed('model', function() {}).volatile()
+    }`,
+  `{
+      foo: computed(function() {})
+    }`,
+  `{
+      foo: computed(async function() {})
+    }`,
+  `{
+      foo: computed('model', async function() {})
+    }`
+];
+
+const codeWithGetters = [
+  `{
+      foo: computed({
+        get() {
+          return true;
+        }
+      }).readonly()
+    }`,
+  `{
+      foo: computed({
+        get() {
+          return true;
+        }
+      })
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {
+          return true;
+        }
+      }).readonly()
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {
+          return true;
+        }
+      })
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {}
+      })
+    }`
+];
+
+const codeWithSettersAndGetters = [
+  `{
+      foo: computed({
+        get() {
+          return true;
+        },
+        set() {
+          return true;
+        }
+      }).readonly()
+    }`,
+  `{
+      foo: computed({
+        get() {
+          return true;
+        },
+        set() {
+          return true;
+        }
+      })
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {
+          return true;
+        },
+        set() {
+          return true;
+        }
+      }).readonly()
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {
+          return true;
+        },
+        set() {
+          return true;
+        }
+      })
+    }`,
+  `{
+      foo: computed('model.foo', {
+        get() {},
+        set() {},
+      })
+    }`
+];
```

Comment:
can you check what happens for something like:

```js
      foo: computed('model.foo', {
        set() {},
      })
```

Reply:
Good catch, I wasn't handling this case well. I'm not sure what *should* happen though. I'm thinking that `set` is always rejected unless it is paired with a `get` no matter what options the user picks? Or is `set` ok on it's own and would basically override the computed property entirely? I don't know how ember would treat this type of property at all.

Reply:
> I'm thinking that set is always rejected unless it is paired with a get no matter what options the user picks?

yeah, I think that's what I would go for too.

@bmish @rwjblue thoughts on this?

Reply:
I've added rules and tests for the setter only scenarios. I didn't squash the commit though in case you would like to see a different way forward.